src/app/components/ATIAnalytics/canonical/sendBeaconOperaMini.test.ts

```diff
@@ -53,6 +55,35 @@ describe('sendBeaconOperaMiniScript', () => {
     );
   });
 
+  it('should NOT send more than 1 beacon with XHR, when browser is Opera Mini', () => {
+    const check = {
+      hasScriptRun: false,
+      operamini: new OperaMiniMock(),
+      get hasOperaMinScriptRun() {
+        return this.hasScriptRun;
+      },
+
+      set hasOperaMinScriptRun(val) {
+        this.hasScriptRun = val;
+      },
```

Comment:
```suggestion
      hasOperaMinScriptRun : undefined,
```
why does the flag need to be implemented with getters + setters in the mock? I would've thought you could have just implemented it as a mutable field?